packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -431,6 +431,7 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: SQSEvent) => {
         const conversionStart = Date.now();
         let conversionResult: FHIRBundle;
         if (isSidechainConnector()) {
+          if (!payloadRaw.length) throw new Error(`Xml document is empty`);
```

Comment:
I suggest we don't throw error... if the XML is empty, there's nothing we can do, right? Assuming it came empty from CommonWell and not an internal error, obviously.

Also, when checking for the lenght, I think we should `.trim()` it first